discovery.md

```diff
@@ -562,12 +572,11 @@ prepared to support paginated responses.
 
 #### `GET /services/{id}`
 
-If this refers to a valid Service, then this MUST return that single Service
-entity.
+This MUST return the Service if it exists.
 
 The following responses are defined by this specification:
 - `200 OK` and the Service representation in the HTTP Response Body.
-- `404 Not Found` if there is no Service with the specified `id`.
+- `404 Not Found` if the Service does not exist.
```

Comment:
it's kind of pedantic, but as it stands now this doesn't actually say which service to return. Meaning, there's no MUST to link the "id" to the service. It is certainly implied though. Perhaps we need to make it explicit

Reply:
Haha.  That really is kind of pedantic.  But, I love it and agree.  Implied, sure, but I can improve the language.

Does `s/the Service if/the latest version of the Service with the given `id` if/ suffice in your opinion?

Also adding the `latest version` language to the `GET /services` endpoint.

Reply:
> Does s/the Service if/the latest version of the Service with the given id` if/ suffice in your opinion?

yes